examples/gno.land/r/demo/markdown_test/markdown.gno

```diff
@@ -175,6 +173,403 @@ If markdown is too limiting, you can just insert your own <strike>crazy</strike>
 <div style='font-family: "Comic Sans MS", "Comic Sans", cursive;'>
 It is a pity, but markdown does **not** work in here for most markdown parsers.
 [Marked] handles it pretty well.
-</div>`
+</div>
+
+Components
+==========
+
+Jumbotron component
+-------------------
+
+A Jumbotron is a layout component used to showcase key messages.
+
+
+:::jumbotron
+
+## Jumbotron title
+
+This is a Jumbotron component you can fill with regular markdown.
+
+:::jumbotron/
+
+
+Stack component
+---------------
+
+Stack is a layout component used to group elements together and apply a space between them. For example, it is usefull with grouped links.
+
+:::stack
+[Stack link 1](/)
+[Stack link 2](/)
+:::stack/
+
+Box component
+-------------
+
+Box is the most abstract component that renders a div element. It is used as a group component.
+
+:::box
+**This content is in a box**
+:::box/
+
+
+Columns component
+-----------------
+
+Columns is a layout for managing columns layouts. It takes the numb of cols you want (*from 1 to 6 columns*) to create as argument and need to be filled with as much content as you mentioned in this argument. You can manage multiple element by grouping them in box components. 
+
+*Nested columns are not currently allowed*
+
+:::columns (2)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::columns/
+
+:::columns (3)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::columns/
+
+:::columns (4)
```

Comment:
```suggestion
:::columns(4)
```

It seems more intuitive to envision keywords as functions that can take zero, one, or multiple parameters.

Reply:
I also agree with you. This is also more compliant with Mardown style.